engine/providers.go

```
@@ -0,0 +1,280 @@
+	vault string
+}
+
+func newOnePasswordProvider() (*onePasswordProvider, bool) {
+	if runtime.GOOS != "linux" {
+		return nil, false
+	}
+	if _, err := exec.LookPath("op"); err != nil {
+		return nil, false
+	}
+	// Smoke test: op is authenticated.
+	out, err := exec.Command("op", "vault", "list", "--format=json").CombinedOutput()
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)